x-pack/plugins/session_view/public/components/session_view/index.tsx

```diff
@@ -124,8 +128,20 @@ export const SessionView = ({
     isFetching: isFetchingAlerts,
     hasNextPage: hasNextPageAlerts,
     error: alertsError,
+    refetch: refetchAlerts,
   } = useFetchSessionViewAlerts(sessionEntityId, investigatedAlertId);
 
+  useEffect(() => {
```

Comment:
can the click handler for the refresh buttton just call refetch/refetchAlerts, unsure why we need the state var + useEffect hook.